rest-api/src/test/resources/test_apm_entity_ids.json

```
@@ -0,0 +1 @@
+["DogCow", "TestApmEntityId"]
```

```suggestion
["A123", "1234-0000"]
```
Would it be better to do something that looks more like an APM entity id here?

---

We could, as these are just test values. The only effect would be resetting the test files to use these. I can go ahead and do that.